examples/BioModels/event.ant

```diff
@@ -0,0 +1,5 @@
+model flux()
+    compartment comp1 = 0;
+    React0: B -> BL; comp1*(kf_0*B - kr_0*BL);
+    React0 >= 0
+end
```

Comment:
This isn't an event model; it's a flux balance model.

Reply:
This was a mistake, this file should not be here